modules/contracts/src.sol/ChannelFactory.sol

```diff
@@ -25,6 +27,11 @@ contract ChannelFactory is IChannelFactory, MinimalProxyFactory {
       return mastercopy;
     }
 
+    /// @dev Allows us to get the chainId that this factory will use in the create2 salt
+    function getChainId() external override view returns(uint) {
+      return chainId;
+    }
+
```

Comment:
Hmm, this is technically totally correct, but I worry that it might be misleading:
It's just a getter for the stored variable and, as such, follows the naming convention. However, the name of the function seems to suggest this returns the actual chain id we're using in the salt computation.
If a function that returns the value stored in the variable is actually useful (in tests, for example, to verify what gets stored), I'd suggest to deviate in this case from the naming convention and name the function `getStoredChainId`, just to avoid any potential confusion.
The `getChainId` function should, in my opinion, return the "real" chain id that is used in the salt computation.